Sources_v3/Query/ChannelListQuery.swift

```diff
@@ -87,27 +87,24 @@ public struct ChannelListQuery<ExtraData: ChannelExtraData>: Encodable {
     /// A number of messages inside each channel.
     public let messagesLimit: Pagination
     /// Query options.
-    public let options: QueryOptions
+    let options: QueryOptions = [.watch]
```

Comment:
Let's make it `var` if it's not public? I think it will be easier to work with like that.